packages/web/src/pages/stats/components/bulletin-card.tsx

```diff
@@ -10,16 +10,28 @@ interface IProps {
   bodyStyle: Record<string, any>;
 }
 
+
 const BulletinCard: React.FC<IProps> = ({ data, type, race, bodyStyle }) => {
   const bulletinData = data["intelBulletins"][race as "soviet"];
+  const [toggle, setToggle] = useState(true)
 
   // Here are all the bulletin data
   // console.log(bulletinData)
 
+  const sliced = Object.fromEntries(
+      Object.entries(bulletinData).slice(0, 20)
+  )
+
   return (
-    <Card title={`Intel Bulletins  ${type} - ${race}`} bodyStyle={bodyStyle}>
-      <BulletinsBarChart bulletins={bulletinData} />
-    </Card>
+
+      <Card
+        title={`Intel Bulletins  ${type} - ${race} `}
+        bodyStyle={bodyStyle}
+        extra = {<button onClick={() => setToggle(!toggle)}> {!toggle ? <span>Display Top 20</span> : <span>Display All Results</span> }</button>}
```

Comment:
Please try to use https://ant.design/components/switch/ this component, it will look better :) By default is should be enabled and say "Display Top 20"